src/main/java/thePackmaster/cards/startuppack/DashIn.java

```diff
@@ -0,0 +1,36 @@
+package thePackmaster.cards.startuppack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.unique.WhirlwindAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class DashIn extends AbstractStartUpCard {
+    public final static String ID = makeID("DashIn");
+
+    public DashIn() {
+        super(ID, -1, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ALL_ENEMY);
+        this.isInnate = true;
+        this.baseDamage = this.damage = 4;
+        this.baseBlock = this.block = 4;
```

Comment:
This seems a little strong due to all the scaling it benefits from. Maybe reduce block gain by 1?

Reply:
sure, but just to check, you realize that the block does not scale with X, but you always just gain it once?

Reply:
Nope, i did not realize. All good then